### DISCUSSION TITLE
New analog type without pointers

### DISCUSSION THREAD

**Discussion by b4sti** [jomjol/AI-on-the-edge-device#1122] [General]
Hi,

I have a meter of  type Allmess AMES 3-K+m A34, which has a weird analog counter. The counter has no written scala on it and no pointer. It is not clear to me, what value it shows, I just found out that it makes one turn per liter. If I record images of it for the training data and just choose the values for each rotation somehow, will it be possible to support it? Or does it mess up the training with other analog counters?
![image](https://user-images.githubusercontent.com/2328127/193461542-ade2eea9-d8f7-4d81-a498-1c096b6529a0.png)

Btw, I you have the choice to buy a meter, avoid this type. The curvature of the glass over the digits makes it especially hard to get rid of reflections. The image above was the best I could get out of it, but sometimes the reflections on the bottom of each digit still messes up the detection.

**Comment by jomjol** [comment]
I would give it a try. Maybe you need a dedicated trained network, if you have some python background, that should not be a problem.

**Reply by b4sti** [reply]
Wow, that was a fast response, thanks. I will collect some training data and post it once ready. Python is not an issue, but I have no background with tensorflow yet. Is there any guideline on how to create own tflite from image files?

**Comment by ghost** [comment]
I would use extendedValue and remove the analog Pointer. It does the same and should be more accurate.

**Reply by b4sti** [reply]
By extendedValue you mean the new intermediate value feature for the last digit? I did not test it yet as I have an older version installed at the moment, but I think it can not be as accurate as the movement of the digit is not continuous, e.g. here:
![image](https://user-images.githubusercontent.com/2328127/193466882-355438da-f497-4bf3-a783-7fc56e092c8b.png)
From my interpretation of the analog counter, the value should be approximately 1.9, while the digit looks like 2.0

**Comment by caco3** [comment]
You also could simply keep the analogue digit away if you are happy with a 1 liter resolution.
